bentoml/cli/__init__.py

```diff
@@ -118,24 +119,29 @@ def cli():
 
     # pylint: disable=unused-variable
 
-    @_cli.command(help='Generate serverless project with BentoML service archive.',
-                  context_settings=dict(ignore_unknown_options=True, allow_extra_args=True))
+    @_cli.command(help='Deploy BentoML archive to AWS Lambda or Google Cloud Function as ' +
+                  'REST endpoint with Serverless Framework')
     @click.argument('archive-path', type=click.STRING)
-    @click.argument('output-path', type=click.STRING)
-    @click.option('--platform', type=click.Choice(['aws-python', 'aws-python3', 'google-python']),
-                  default='aws-python3')
-    @click.pass_context
-    def build_serverless_archive(ctx, archive_path, output_path, platform):
-        if which('serverless') is None:
-            click.echo('Serverless framework is not installed', err=True)
-            click.echo('Please visit www.serverless.com for install instructions')
+    @click.option('--platform', type=click.Choice([
+        'aws-lambda', 'aws-lambda-py2', 'gcp-function', 'aws-sagemaker', 'azure-ml', 'algorithmia'
+    ]), required=True)
+    @click.option('--region', type=click.STRING)
+    @click.option('--stage', type=click.STRING)
+    def deploy(archive_path, platform, region, stage):
+        if platform in SERVERLESS_PLATFORMS:
+            bento_service = load(archive_path)
```

Comment:
put this inside `deploy_with_serverless `?

Reply:
done